Source/Scene/Cesium3DTileset.js

```diff
@@ -2777,8 +2802,13 @@ function update(tileset, frameState, passStatistics, passOptions) {
     if (defined(credits) && statistics.selected !== 0) {
       const length = credits.length;
       for (let i = 0; i < length; ++i) {
-        frameState.creditDisplay.addCredit(credits[i]);
+        const credit = credits[i];
+        if (this._showCreditsOnScreenDirty) {
+          credit.showOnScreen = this._showCreditsOnScreen;
```

Comment:
Use `tileset` since this is in a local function

```suggestion
          credit.showOnScreen = tileset._showCreditsOnScreen;
```